parsers/toml/toml_test.go

```diff
@@ -0,0 +1,142 @@
+package toml
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTOML_Unmarshal(t *testing.T) {
+
+	testCases := []struct {
+		name   string
+		input  []byte
+		keys   []string
```

Comment:
Instead of `keys, values`; it might be better to expect a `map`?

Reply:
And then how do we plan to assert them? using `reflect`?

Reply:
The testify lib can handle this. Checkout https://github.com/knadh/koanf/blob/master/parsers/dotenv/dotenv_test.go#L137

Reply:
fixed